reports/performance/_model.qmd

```diff
@@ -1258,3 +1258,126 @@ model_big_misses_assessment %>%
 ```
 
 :::
+
+## Variance Over Time
+
+These charts look at the variance of the sale price and estimated FMV over time. Ideally, these should remain stable. One should be aware that while ratios may demonstrate consistent trends, because housing values tend to increase over time, the aggregate difference may increase.
+
+::: {.panel-tabset}
+
+
+```{r _organize_variance_data}
+training_data_monthly <- training_data_pred %>%
+  filter(!ind_pin_is_multicard, !sv_is_outlier) %>%
+  mutate(
+    meta_sale_date = as.Date(meta_sale_date),
+    year = year(meta_sale_date),
+    month = month(meta_sale_date),
+    difference = (pred_card_initial_fmv - meta_sale_price),
+    squared_difference = difference^2
+  ) %>%
+  group_by(year, month) %>%
+  summarize(
+    total_sales = sum(meta_sale_price),
+    total_fmv = sum(pred_card_initial_fmv),
+    variance_sale = var(meta_sale_price),
+    variance_fmv = var(pred_card_initial_fmv),
+    mean_difference = mean(difference),
+    sse = sum(squared_difference),
+    n = n(),
+    .groups = "drop"
+  ) %>%
+  mutate(
+    variance_diff = variance_fmv - variance_sale,
+    date = make_date(year, month),
+    variance_ratio = variance_fmv / variance_sale,
+    percent_sales = n / sum(n) * 100,
+    percent_sse = sse / sum(sse) * 100
+  )
+
+training_data_monthly_long <- training_data_monthly %>%
+  pivot_longer(
+    cols = c(variance_sale, variance_fmv, percent_sales, percent_sse, variance_diff),
+    names_to = "Metric",
+    values_to = "Value"
+  )
+```
+
+### Variance Ratio (FMV / Sale Price)
+
+```{r _variance_ratio_chart}
+ggplot(training_data_monthly, aes(x = date, y = variance_ratio)) +
+  geom_line() +
+  geom_point() +
+  labs(
+    x = "Date",
```

Comment:
**suggestion:** Let's make all the x-axis labels "Date" instead of "Month" (as below).